db/statistics.go

```diff
@@ -119,44 +119,51 @@ func WriteValidatorStatisticsForDay(day uint64) error {
 	}
 
 	batchSize := 16000 // max parameters: 65535
-	for b := 0; b < len(incomeStats); b += batchSize {
```

Comment:
This fixes the second bug that "Yesterday" income does not get set (and is then also missing in "Last Week" & "Last Month") for validators that are too new.
This is because **incomeStats** is a map and not a slice and does not contain every validator.